agent/src/main/resources/reference.conf

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# dispatcher to be used (http or console)
+atum.dispatcher.type="http"
+
+# The REST API URI of the atum server
+#atum.dispatcher.http.url=
```

Comment:
I would even say let's use it and with this value: http://localhost:8080, what do you think?

Reply:
Let's say I run Atum in production and forgot to specify the url. The localhost will be used, and it will fail only when the data are being sent, whereas now it fails during initialization. In this case I think it's good to fail fast.

But I think I could add `application.conf.template` so every developer can just copy it and modify it as they please.

Reply:
I added the template